### ISSUE TITLE
Edge: button functionality is lost when increasing screen size to 200%

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#125]
URL: Any page
User Agent: Edge
Language: Both

Issue Description: In Edge, when the screen is zoomed to 200% some buttons lose their functionality when trying to access them via the mouse.

Steps to reproduce:
1.  From Edge, select the ellipsis in the top right hand corner and change Zoom to 200% 
2. Click on the 'Report a problem or mistake on this page' button
3. Using the mouse, click on the 'Share this page' button
4.  Neither button is functional 
![image](https://user-images.githubusercontent.com/68117281/137519483-6fdea630-a614-4bfc-89f8-a321e3a5f2a0.png)

Expected results: No functionality should be lost when increasing screen size to 200%

**Comment by echouin** [comment]
@ndolter from OP (a video is also included at the link):

I was unable to duplicate this problem. I followed the steps above and it worked for me, using Edge on Windows 10, 2015 surface pro.
﻿
﻿see attached﻿﻿ video: https://openplus.monday.com/boards/947494094/views/35132233/pulses/1793938687/posts/1198054701#:~:text=attached%EF%BB%BF%EF%BB%BFscreen-recorder-fri-oct-15-2021-13-01-59.webm

﻿download the file and it should open and play in edge

**Comment by ndolter** [comment]
@echouin , I'm running Edge ver 94.0 in Win 10.  My co-worker Saratha also has the same issue as me.

**Comment by echouin** [comment]
@ndolter from OP:
I can't duplicate this issue in a real world situation.
1. I tried windows 10 edge 94 on surface pro > success (as per the video I sent)
2. I tried widnows 10, edge 94 desktop > success
3. I tried edge 94 on OSX > success
3. I tried an emulator at saucelabs and was able to duplicate the problem there, this isn't a real world situation though

I'm really not sure how to proceed with this. I can't fix what I can't duplicate.

**Comment by echouin** [comment]
@ndolter please let me know next steps?

**Comment by mmiron-HC** [comment]
@echouin we do not have access to Monday.com.
Without access to the RSAMS space we can no longer validate.

**Comment by echouin** [comment]
@mmiron-HC sent video from OP to your work email.

**Comment by echouin** [comment]
@mmiron-HC this is a public-facing issue. Can you please verify?

OP is unable to verify the issue/error.

**Comment by ndolter** [comment]
Validated in French and English and this has now been resolved.

**Comment by ndolter** [comment]
Re-opening as the issue with the inability to use the mouse in Edge zoomed into 200% is an issue today.  Our other tester Xiu experienced the same thing.  The video I took is too large to attach here.  If required, please let me know and I'll see what else I can do to illustrate the issue.

**Comment by echouin** [comment]
Hi Manon,

I noticed that I’ve been added to this one again.

I had attached the Video that OP provided.

Thank you,

Eric

From: Chouinard, Eric (HC/SC)
Sent: 2021-10-18 8:38 AM
To: mmiron-HC/WST-RSAMS ***@***.***>; mmiron-HC/WST-RSAMS ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [mmiron-HC/WST-RSAMS] Edge: button functionality is lost when increasing screen size to 200% (#125)

Hello Manon,

Here is the video from OP that I downloaded showing it functioning (attached).

Thank you,

Eric

**Comment by davidgotlieb** [comment]
Is there a specific page where this is happening @ndolter . I can't replicate the issue in Edge either, it's working fine for me.

Perhaps hopping on a Teams meeting where we can share screens would help figure out the problem?

**Comment by ndolter** [comment]
Hi @davidgotlieb, the page link is https://recalls-rappels.canada.ca/en.  Manon sent the video  to you and Eric via MS Teams.

**Comment by echouin** [comment]
@ndolter & @mmiron-HC I just went into Edge, enlarged to 200% and had no issues clicking the two buttons: "Report a Problem" and "share this page".

For this reason, I am marking as "resolved".

**Comment by hydegeoff** [comment]
Issue Partially fixed, **Report a problem...** button is working properly, but **Share this page** button is not at 200% tested in FR and EN for both buttons.  Tested using Windows 10 - Microsoft Edge Version 105.0.1343.50 (Official build) (64-bit)

**Comment by echouin** [comment]
Re-tested again on Production website. Everything is working correctly. Recommend to close.

**Comment by zimw7** [comment]
Issue fixed. Both English and French buttons now function correctly.
However, if the "Report a problem or mistake" button is clicked and the menu is open, clicking "Share this page" causes overlap. This should be raised as a separate issue.
![image](https://github.com/user-attachments/assets/a5271e6b-1476-4ecf-9074-8079cd89bcf4)

**Comment by 40999221** [comment]
Issue fixed. Buttons functionally works at 200% too for both English and French.

![image](https://github.com/user-attachments/assets/480a6def-a33f-4d90-9d52-541620361418)

![image](https://github.com/user-attachments/assets/140d3b18-9c58-494d-90fd-80f318b48fe6)

![image](https://github.com/user-attachments/assets/e4f31ebf-1d9c-4a4a-a6d3-dcbb594bee73)

**Comment by YuxuanXie01** [comment]
Retested, issue fixed. However, I am concerned about a related problem: when you click "Report a problem or mistake on this page" to activate the dropdown menu and then select "Share this page", the two windows overlap when the page is zoomed to 200%.
![image](https://github.com/user-attachments/assets/dd87e576-e4bd-4961-a11f-d09252625343)
